src/os/bluestore/BlueFS.cc

```diff
@@ -271,13 +275,29 @@ void BlueFS::_init_logger()
   b.add_u64_counter(l_bluefs_read_random_count, "read_random_count",
 		    "random read requests processed");
   b.add_u64_counter(l_bluefs_read_random_bytes, "read_random_bytes",
-		    "Bytes requested in random read mode", NULL,
+		    "Bytes requested in random read mode",
+		    NULL,
 		    PerfCountersBuilder::PRIO_USEFUL, unit_t(UNIT_BYTES));
   b.add_u64_counter(l_bluefs_read_random_disk_count, "read_random_disk_count",
 		    "random reads requests going to disk");
   b.add_u64_counter(l_bluefs_read_random_disk_bytes, "read_random_disk_bytes",
-		    "Bytes read from disk in random read mode", NULL,
+		    "Bytes read from disk in random read mode",
+		    "rrd",
```

Comment:
I've noticed that you omit the `b` for `bytes` a lot. Are you implying that this is sort of the default nick? I would consider `rrdb` in case someone needs to set the nick for `read_random_disk_count` to `rrdc`, but I can see if this is meant to imply that this counter has the default nick.